telegram/ext/jobqueue.py

```diff
@@ -264,7 +260,8 @@ def tick(self):
 
             if job.enabled:
                 try:
-                    current_week_day = datetime.datetime.now().weekday()
+                    # TODO: set timezone for checking weekday
```

Comment:
I would note that making the timezone configurable per `Job` is important. Being able to set a default in the `JobQueue` might be nice, but i don't think it's essential.
Do you think adding this is still in the scope of this PR?

Reply:
> I would note that making the timezone configurable per `Job` is important.
> Do you think adding this is still in the scope of this PR?

Yep, I've just pushed the commit.

Reply:
Awesome! I think it would be great if we had a test case like `test_run_daily` but with a timezone-aware time. Would you mind adding that here as well, to prevent future regressions in regards to timezone handling?

Reply:
> I think it would be great if we had a test case like `test_run_daily` but with a timezone-aware time.

I included that in the same commit; it's a bit flaky though, suggestions for improvement are welcome :wink:

Reply:
Hmm that test seems to only test for the correct selection of the weekday, while `test_run_daily` compares the specified time specifically. One might imply the other but as i said, timezones confuse me ^^